openshift-with-appstudio-test/e2e/util.go

```diff
@@ -189,7 +189,7 @@ func commonSetup(t *testing.T, gitCloneUrl string, namespace string) *testArgs {
 		debugAndFailTest(ta, err.Error())
 	}
 
-	mavenYamlPath := filepath.Join(path, "..", "..", "deploy", "base", "maven-v0.2.yaml")
+	mavenYamlPath := filepath.Join(path, "..", "..", "deploy", "base", "config", "maven-v0.2.yaml")
```

Comment:
This should actually be moved to hack, it is only used for the tests/examples.

Reply:
Are you talking about `gradle-v0.1.yaml` and`maven-v0.2.yaml` (and there is also a config.yaml in that directory currently used to enableRebuilds and set some mavenBaseLocations) ? All of these are configured as well by the dev-template directory to patch the image used - while I see the e2e tests patch the image on about line 209, from what I can see the examples do not patch the image or load the task themselves?

Reply:
I've moved the 3 file referenced and modified the exampes to patch the image according to `QUAY_USERNAME`.